pkgs/by-name/si/simplesamlphp/package.nix

```diff
@@ -0,0 +1,47 @@
+{ php
+, fetchFromGitHub
+, lib
+}:
+php.buildComposerProject (finalAttrs: {
+  pname = "simplesamlphp";
+  version = "1.19.7";
+
+  src = fetchFromGitHub {
+    owner = "simplesamlphp";
+    repo = "simplesamlphp";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-Qmy9fuZq8MBqvYV6/u3Dg92pHHicuUhdNeB22u4hwwA=";
+  };
+
+  php = php.buildEnv {
```

Comment:
Why do you need a custom version of PHP ?

Reply:
SimpleSAMLphp requires PHP extensions, and as I understand, it requires custom version of PHP.

Reply:
The default PHP version already includes a comprehensive set of extensions, see: https://github.com/NixOS/nixpkgs/blob/d5f39744c87dbbfe55a1d8ae2da409293b2d2760/pkgs/development/interpreters/php/8.1.nix#L17

Isn't this enough?

Reply:
or I am wrong?

Reply:
I'm asking you...

Reply:
Oh, that's enough. I will delete phpEnv then.

Reply:
I think you can delete the `php` attribute indeed.

Reply:
I deleted it. Thank you!